backend/routes/routes.js

```diff
@@ -17,6 +18,9 @@ router.post('/inventory', authentication, isAdmin, validateProduct, createProduc
 router.put('/inventory/:id', authentication, isAdmin, validateProduct, updateProduct)
 router.delete('/inventory/:id', authentication, isAdmin, deleteProduct)
 
+// product quantity update routes
+router.put('/product/increase/:id', authentication, validateQuantity, receiveProduct)
```

Comment:
Nit pick: I feel `/product/:id/increase` is better use for this case. API resource should be identify before the command. Other than that, if you need classic REST API, we can use `PATCH /product/:id` and use below body to control operation

```json
{ 
    "quantity": 5, 
    "operation": "increase"
}
```